superset/reports/notifications/email.py

```diff
@@ -70,9 +70,11 @@ def _get_content(self) -> EmailContent:
             image = {msgid: self._content.screenshot.image}
             body = __(
                 """
+                <p>%(description)s</p>
```

Comment:
Is the description just for emails?

Reply:
that was the ask. Should I also add the description to slack notifications while I'm at it?

Reply:
@junlincc ^^^

Reply:
Yes, let's add description to slack notifs as well.